raiden/network/transport/matrix/client.py

```diff
@@ -244,6 +244,31 @@ def __init__(
         # can remain constant over multiple loops (if no events occur).
         self._sync_iteration = 0
 
+        self._sync_filter_id: Optional[int] = None
+
+    def create_sync_filter(self, broadcast_rooms: Dict[str, Optional[Room]]) -> None:
+        ignore_rooms = [room.room_id for room in broadcast_rooms.values() if room is not None]
+        filter_params = {
+            "room": {
+                # Filter out all room state events / messages from broadcast rooms
+                "not_rooms": ignore_rooms,
+                # Ignore "message recipts" from all rooms
+                "ephemeral": {"not_types": ["m.receipt"]},
+            },
+            # Get all presence updates
+            "presence": {"types": ["m.presence"]},
+        }
+
+        filter_id = None
+        try:
+            # 0 is a valid filter ID
+            filter_response = self.api.create_filter(self.user_id, filter_params)
```

Comment:
Do we need to clean up the created filters somehow?

Reply:
I noticed that if we're sending the same filter payload, the server would just reuse our existing filter.

Reply:
I will add an assumption test for this.